### PR TITLE
[TR] Collab Information

### PR DESCRIPTION
**Pull request by Lxran**
Added the Turkish translation of the "Collab Information" article.

### CODE REVIEW

File: wiki/Guides/Collab_Information/tr.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Beatmap Yaratıcıları
+
+İşte burası herkesin aklını kaybettiği yer. Hayalini kurduğunuz beatmap yaratıcılarını yanınıza almak pek kolay olmayabilir ancak olur da alırsanız, başlarken her açıdan hazır olmalısınız.
+
+- Kendi beatmap yaratma yeteneklerinizi geliştirin. Anahtar noktalardan biridir. Eğer işbirliği yapacağınız kişi sizin beatmap yapımınızı beğenmezse... onların arkasından su dökseniz iyi olur.
+- Kendinize benzer stilde beatmap yaratan birini seçin. Benzer akışa sahip olduğunuzda daha kolay olur.
+- Birbirinizle kolaylıkla iletişime geçebileceğinizden emin olun... **Bu önemli çünkü beraber çalıştığınız kişiler herhangi bir sebepten ötürü kaybolurlarsa, tek başınızasınız.**
+
+Unutmayın, iyi bir beatmapin zaman alan ve potansiyelinin zirvesinde olan olduğunu aklınızda tutun. İşbirlikleri sıkça derecli olması en zor olan beatmaplerdir. Sebeplerinden birkaçı şunlardır:
```
Comment (Drops-P) [inline wiki/Guides/Collab_Information/tr.md:49]:
```suggestion
İyi bir beatmapin zaman aldığını ve potansiyelinin zirvesinde olduğunu aklınızdan çıkartmayın. İşbirlikleri, sıkça dereceli olması en zor olan beatmaplerdir.  Sebeplerinden birkaçı şunlardır:
```

File: wiki/Guides/Collab_Information/tr.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Kolaylıkla göz ardı edilebilen, edildiğinde tehlikeli olan bir adım.
+
+- Hepinizin ortak karar kılabileceği bir şarkı seçin.
+- Zamanlama her zaman önemlidir.
+- İşbirlikleri için genellikle müziğinde belirgin değişiklikleri olan ya da belirgin parçaları olan şarkılar seçilir.
+
+## İş bölümü
+
+Diğer kişinin/kişilerin belirgin bir şekilde yaratmaya başlayabileceği parçaların olması çoğunlukla iyidir.
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:27]:
```suggestion
Birlikte çalıştığınız diğer kişinin/kişilerin rahatça maplemeye başlayabileceği parçalara sahip olmak çoğunlukla iyidir.
```
flows better imo

File: wiki/Guides/Collab_Information/tr.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- Öncelikle, şarkının kendinizin olarak belirlediğiniz kısmına yer işaretleri bırakmanız tavsiye edilir. Tercihen olabildiğince eşit bir şekilde.
+- Beatmapi yaratmaya kim başlarsa başlasın, beatmapinizin kendi kısımlarınızı tek seferde yapmaktan ziyade baştan sona ilerlemek çok daha kolaydır. Böylece akışı ve oynanışı görmek çok daha kolaydır.
+- Boşlukları hesaba katmayı aklınızda tutun. **Pek çok kişi bunu unutur.**
+
+## Beatmap Ayarları
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:33]:
```suggestion
## Map Ayarları
```

File: wiki/Guides/Collab_Information/tr.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Eğer beatmap ayarlarında karar kılamazsanız... canınız çok yanacak.
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:35]:
```suggestion
Eğer map ayarlarında karar kılamazsanız...canınız çok yanacak.
```

File: wiki/Guides/Collab_Information/tr.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+- Slider Hızında karar kılın. Bu Last Time Travel ve Gonna Be Here için çok acı bitti.
+- Mesafe aralığına ve kilidinin kullanımı hakkında ortak karar alın. [Be Your Wings](https://osu.ppy.sh/beatmapsets/8518) beatmapine bakın.
+- Zorluk seçeneklerine ortak karar kılın.
+
+## Beatmap Yaratıcıları
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:41]:
```suggestion
## Mapperlar
```

File: wiki/Guides/Collab_Information/tr.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+İşte burası herkesin aklını kaybettiği yer. Hayalini kurduğunuz beatmap yaratıcılarını yanınıza almak pek kolay olmayabilir ancak olur da alırsanız, başlarken her açıdan hazır olmalısınız.
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:43]:
```suggestion
İşte burası herkesin aklını kaybettiği yer. Hayalini kurduğunuz mapperları yanınıza almak pek kolay olmayabilir ancak olur da alırsanız, başlamadan önce her açıdan hazır olmalısınız.
```

File: wiki/Guides/Collab_Information/tr.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+- Kendi beatmap yaratma yeteneklerinizi geliştirin. Anahtar noktalardan biridir. Eğer işbirliği yapacağınız kişi sizin beatmap yapımınızı beğenmezse... onların arkasından su dökseniz iyi olur.
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:45]:
```suggestion
- Kendi mapleme yeteneklerinizi geliştirin. Anahtar noktalardan biridir. Eğer işbirliği yapacağınız kişi sizin nasıl maplediğinizi beğenmezse... İşbirliğine elveda diyebilirsiniz.
```

File: wiki/Guides/Collab_Information/tr.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- Kendinize benzer stilde beatmap yaratan birini seçin. Benzer akışa sahip olduğunuzda daha kolay olur.
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:46]:
```suggestion
- Kendinize benzer stilde mapleyen birini seçin. Maplerken benzer akışa sahip olmak sıklıkla işi kolaylaştırır.
```
flows better

File: wiki/Guides/Collab_Information/tr.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+- Birbirinizle kolaylıkla iletişime geçebileceğinizden emin olun... **Bu önemli çünkü beraber çalıştığınız kişiler herhangi bir sebepten ötürü kaybolurlarsa, tek başınızasınız.**
+
+İyi bir beatmapin zaman aldığını ve potansiyelinin zirvesinde olduğunu aklınızdan çıkartmayın. İşbirlikleri, sıkça dereceli olması en zor olan beatmaplerdir.  Sebeplerinden birkaçı şunlardır:
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:49]:
```suggestion
İyi bir mapin zaman aldığını ve potansiyelinin zirvesinde olduğunu aklınızdan çıkartmayın. İşbirlikleri, sıkça dereceli olması en zor olan maplerdir. Sebeplerinden birkaçı şunlardır:
```

File: wiki/Guides/Collab_Information/tr.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+- Birden fazla beatmap yaratıcısı ile çalışıyorsunuzdur, ve eğer modlayıcılardan (modderlardan) birisi o yaratıcının bölümünde bir hata bulursa, o yaratıcı bunu çözmek zorundadır.
```
Comment (zeusminus) [inline wiki/Guides/Collab_Information/tr.md:51]:
```suggestion
- Birden fazla mapper ile çalışıyorsunuzdur, ve eğer modlayıcılardan (modderlardan) birisi o mapperın bölümünde bir hata bulursa, o mapper bunu çözmek zorundadır.
```

**Pull request merged**